es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/clients/HighLevelClientBasedIndicesAdminClient.scala

```diff
@@ -0,0 +1,270 @@
+
+  override def prepareShardStores(indices: String*): IndicesShardStoreRequestBuilder = ???
+
+  override def create(request: CreateIndexRequest): ActionFuture[CreateIndexResponse] = ???
+
+  override def create(request: CreateIndexRequest, listener: ActionListener[CreateIndexResponse]): Unit = ???
+
+  override def prepareCreate(index: String): CreateIndexRequestBuilder = ???
+
+  override def delete(request: DeleteIndexRequest): ActionFuture[AcknowledgedResponse] = ???
+
+  override def delete(request: DeleteIndexRequest, listener: ActionListener[AcknowledgedResponse]): Unit = ???
+
+  override def prepareDelete(indices: String*): DeleteIndexRequestBuilder = ???
+
+  override def close(request: CloseIndexRequest): ActionFuture[CloseIndexResponse] = ???
+
+  override def close(request: CloseIndexRequest, listener: ActionListener[CloseIndexResponse]): Unit = ???
+
+  override def prepareClose(indices: String*): CloseIndexRequestBuilder = ???
+
+  override def open(request: OpenIndexRequest): ActionFuture[OpenIndexResponse] = ???
+
+  override def open(request: OpenIndexRequest, listener: ActionListener[OpenIndexResponse]): Unit = ???
+
+  override def prepareOpen(indices: String*): OpenIndexRequestBuilder = ???
+
+  override def refresh(request: RefreshRequest): ActionFuture[RefreshResponse] = ???
+
+  override def refresh(request: RefreshRequest, listener: ActionListener[RefreshResponse]): Unit = ???
+
+  override def prepareRefresh(indices: String*): RefreshRequestBuilder = ???
+
+  override def flush(request: FlushRequest): ActionFuture[FlushResponse] = ???
+
+  override def flush(request: FlushRequest, listener: ActionListener[FlushResponse]): Unit = ???
+
+  override def prepareFlush(indices: String*): FlushRequestBuilder = ???
+
+  override def syncedFlush(request: SyncedFlushRequest): ActionFuture[SyncedFlushResponse] = ???
+
+  override def syncedFlush(request: SyncedFlushRequest, listener: ActionListener[SyncedFlushResponse]): Unit = ???
+
+  override def prepareSyncedFlush(indices: String*): SyncedFlushRequestBuilder = ???
+
+  override def forceMerge(request: ForceMergeRequest): ActionFuture[ForceMergeResponse] = ???
+
+  override def forceMerge(request: ForceMergeRequest, listener: ActionListener[ForceMergeResponse]): Unit = ???
+
+  override def prepareForceMerge(indices: String*): ForceMergeRequestBuilder = ???
+
+  override def upgrade(request: UpgradeRequest): ActionFuture[UpgradeResponse] = ???
+
+  override def upgrade(request: UpgradeRequest, listener: ActionListener[UpgradeResponse]): Unit = ???
+
+  override def prepareUpgradeStatus(indices: String*): UpgradeStatusRequestBuilder = ???
+
+  override def upgradeStatus(request: UpgradeStatusRequest): ActionFuture[UpgradeStatusResponse] = ???
+
+  override def upgradeStatus(request: UpgradeStatusRequest, listener: ActionListener[UpgradeStatusResponse]): Unit = ???
+
+  override def prepareUpgrade(indices: String*): UpgradeRequestBuilder = ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)